### DISCUSSION TITLE
发现了一个问题

### DISCUSSION THREAD

**Discussion by Syuu614** [YunYouJun/valaxy#340] [Q&A]
<img width="1243" alt="截屏2024-02-04 16 57 22" src="https://github.com/YunYouJun/valaxy/assets/57835875/7290721c-dbbd-4b68-8ded-828739de26c4">
为什么会出现这个问题？

**Comment by YunYouJun** [comment]
请提供可复现仓库。

**Reply by Syuu614** [reply]
https://github.com/Syuu614/valaxy-blog.git

**Reply by YunYouJun** [reply]
无法打开

**Reply by Syuu614** [reply]
抱歉，仓库目前已设置为公开模式。

**Reply by YunYouJun** [reply]
可以尝试最新版本 `0.18.1`，不会复现该问题。
